### PR TITLE
Fix flyout scrollbar reposition edge case

### LINKED ISSUE #4692: Flyout scrollbar does not reposition properly on zoom
**Describe the bug**
Flyout scrollbar in RTL is not repositioning itself correctly when a user zooms in.

**To Reproduce**

1. Go to the multi_playground
2. Open the flyout in **RTL, Vertical, Start** or **LTR, Vertical, End**
3. Zoom in or out on the web page
Notice how the scrollbar is positioned too far to the left or right.

**Expected behavior**
Scrollbar should be correctly positioned on top of the flyout.

**Screenshots**
Screen Shot 2021-03-08 at 3 13 11 PM

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Stack Traces**

```
Replace with error stack trace.
```

**Additional context**
This for some reason only happens in multi_playground and not in the regular playground.

### PR DESCRIPTION
**Pull request by moniika**
## The basics

- [x] I branched from develop
- [x] My pull request is against develop
- [x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### Resolves

Fixes #4692

### Proposed Changes

Adds `setPosition` call inside of Flyout code to fix issues cause by scrollbar not repositioning correctly in certain edge cases.

#### Behavior Before Change

Scrollbar was not repositioning correctly, as described in #4692

#### Behavior After Change

Scrollbar positions correctly.

### Reason for Changes

Bugfix

### Test Coverage

Manual testing in playground.

 * Desktop Chrome

### Additional Information

This `setPosition` call was removed in #4607 because it was incorrectly assumed to be unnecessary. A comment has been added to reference why it's needed and the bug for additional repro of what it fixes.

### CODE REVIEW

File: core/flyout_base.js [hunk 1]
```diff
@@ -441,6 +441,16 @@ Blockly.Flyout.prototype.positionAt_ = function(width, height, x, y) {
...
+    // If origin changed and metrics haven't changed enough to trigger
+    // reposition in resize, we need to call setPosition. See issue #4692.
+    if (scrollbar.hScroll) {
+      scrollbar.hScroll.setPosition(
+          scrollbar.hScroll.position.x, scrollbar.hScroll.position.y);
+    } else if (scrollbar.vScroll) {
+      scrollbar.vScroll.setPosition(
+          scrollbar.vScroll.position.x, scrollbar.vScroll.position.y);
+
+    }
```
Comment (alschmiedt) [inline core/flyout_base.js:453]: What happens if you have both horizontal and vertical scrollbars? Does it not have to reposition?

**Review by alschmiedt — APPROVED** [review]

File: core/flyout_base.js (hunk 1 above)
Comment (moniika) [reply core/flyout_base.js:453]: That's a good point. I assumed there would only be single direction scrollbars in flyouts, but I can easily change this to support flyouts with paired scrollbars too.
